ipaserver/secrets/kem.py

```diff
@@ -216,6 +235,24 @@ def generate_keys(self, servicename):
         ldapconn.set_key(KEY_USAGE_SIG, principal, pubkeys[0])
         ldapconn.set_key(KEY_USAGE_ENC, principal, pubkeys[1])
 
+    def remove_server_keys(self):
+        """Remove keys from LDAP and disk
+        """
+        self.remove_keys('host')
+
+    def remove_keys(self, servicename):
+        """Remove keys from LDAP and disk
+        """
+        principal = '%s/%s@%s' % (servicename, self.host, self.realm)
+        ldapconn = KEMLdap(self.ldap_uri)
+        ldapconn.del_key(KEY_USAGE_SIG, principal)
+        ldapconn.del_key(KEY_USAGE_ENC, principal)
+        try:
+            os.unlink(self.config['server_keys'])
+        except OSError as e:
+            if e.errno != errno.ENOENT:
+                raise
```

Comment:
Might be more consistent to use installutils.remove_file(), perhaps extend that to grok ENOENT.

Reply:
``installutils`` is for ``ipaserver.install`` only. The function deliberately re-raises other errors, e.g. when the file cannot be removed.